Admin/AbstractAdmin.php

```diff
@@ -2843,6 +2838,25 @@ public function getDashboardActions()
     }
 
     /**
+     * Setting to true will enable mosaic button for
+     * the admin screen.
```

Comment:
You are allowed to go up to 120 characters, you know ;) . If you want to cut before, cut at 80.